sktm/executable.py

```diff
@@ -47,6 +49,9 @@ def setup_parser():
     parser.add_argument("--jurl", help="Jenkins URL")
     parser.add_argument("--jlogin", help="Jenkins login")
     parser.add_argument("--jpass", help="Jenkins password")
+    parser.add_argument("--jretry",
+                        help="Counter to retry Jenkins, default to 30",
+                        type=int, default=30)
```

Comment:
Why use a numeric default here instead of the added constant?

Reply:
Sorry I forgot to replace the number with the added constant. Now default = 30 is removed and help string is updated.